notebooks/1.Getting_Data_Ready.ipynb

```diff
@@ -367,11 +384,16 @@
     "    ]\n",
     "}\n",
     "\n",
-    "create_role_response = iam.create_role(\n",
-    "    RoleName = role_name,\n",
-    "    AssumeRolePolicyDocument = json.dumps(assume_role_policy_document)\n",
-    ")\n",
-    "\n",
+    "try:\n",
+    "    create_role_response = iam.create_role(\n",
+    "        RoleName = role_name,\n",
+    "        AssumeRolePolicyDocument = json.dumps(assume_role_policy_document)\n",
+    "    )\n",
+    "    role_arn = create_role_response[\"Role\"][\"Arn\"]\n",
+    "except iam.exceptions.EntityAlreadyExistsException:\n",
+    "    print(\"The role \" + role_name + \" exists, ignore to create it\")\n",
+    "    role_arn = boto3.resource('iam').Role(role_name).arn\n",
+    "    \n",
     "# AmazonPersonalizeFullAccess provides access to any S3 bucket with a name that includes \"personalize\" or \"Personalize\" \n",
```

Comment:
Says Personalize

Reply:
Found master branch also says Personalize but will fix this.

Reply:
Updating comment line:
> # Attaching AmazonForecastFullAccess to access all actions for Amazon Forecast

Reply:
Addressed by the latest commit.